addons/account/models/account_move.py

```diff
@@ -1405,50 +1405,53 @@ def _compute_payments_widget_reconciled_info(self):
             else:
                 move.invoice_payments_widget = json.dumps(False)
 
+    def _compute_move_taxes_by_group(self, tax_lines):
```

Comment:
@qdp-odoo Could you live with this small refactoring?  (it is just an extra method)

Reply:
Hi @jco-odoo do you know if we can ping someone else to check or maybe  @qdp-odoo you have time to check on this
We have a bug ticket related to this internally.
Thanks!
Best and Happy Friday!
cc' @ren-odoo